src/site/content/en/blog/css-marker-pseudo-element/index.md

```diff
@@ -0,0 +1,302 @@
+---
+title: Custom Bullets with CSS ::marker
+subhead: It is now trivial to customize the color, size or type of number or bullet when using a <ul> or <ol>. 
+authors:
+  - adamargyle
+  - loirooriol
+description: It is now trivial to customize the color, size or type of number or bullet when using a <ul> or <ol>.
```

Comment:
```suggestion
description: It is now trivial to customize the color, size or type of number or bullet when using a <ul> or <ol>.
tags:
  - blog
  - css
```
We need `blog` for this to show up on web.dev/blog

Other relevant tags can be found at https://github.com/GoogleChrome/web.dev/blob/master/src/site/_data/tagsData.json

FYI @rachelandrew